templates/include/telemetry/coreCommon.2.schema.json

```diff
@@ -47,4 +47,7 @@
   "items": {
     "type": "string"
   }
+},
+"bug_1501329_affected": {
```

Comment:
Do we want to add this to all versions of the core ping? Or just the latest?

Reply:
We'll do all